.infrastructure/deployment.yml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: todoapp
+  namespace: mateapp
+spec:
+  strategy:
```

Comment:
Check description to task 
`Resource requests and limits (in idle state, you should have 2 pods running)`